migrations/007.do.add-daily-job-stats.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE daily_stations
+ADD COLUMN honest_measurement_count INT
```

Comment:
Works for me. The downside is that we'll have a bunch of old entries that say 0 counts in the table, which isn't technically correct

Reply:
Yeah, I agree it's not technically correct. At the same time, this applies only to historical data, right? And we can backfill these values if needed.

Let me ask a different question. If we keep NULL values in this column, how do you expect consumers of this data to handle the NULL value?
- Grafana charts
- 3rd-party API consumers

Reply:
Yes, specifically, this applies to "historical data" in the context of "after we released the station_id addition but before we release this PR"

I believe in Grafana you can just use a data transformation for NULL values to treat them as 0. For 3rd party API consumers, we would let them handle it.

I don't have a hard opinion either way though, just devil's advocate here

Reply:
I see your point, and I do value accuracy in our data and DB schema design.

However, in this case, I prefer to prioritise the ease of use for data consumers because the NULL-vs-0 distinction is an edge case in historical data, and nobody will care about that fine detail very soon (once we start recording real data).